rovilay/ideazz-coalition
# Initialize app server

#### What does this PR do?
This PR covers the tasks outlined below 👇🏽

#### Description of Task to be completed?
- initialize server
- setup routes
- setup errorHandler middleware
- configure eslint
- configure es6 with babel

#### How should this be manually tested?
- git clone this app - `git clone https://github.com/rovilay/ideazz-coalition.git`
- navigate to the cloned app
- git checkout to this branch - `git checkout -b ch-setup-server`
- set up the `.env` with the `.env.sample` file
- install dependencies - `npm install`
- start the app - `npm run server`
- navigate to `localhost:port` on the browser
- you should see a welcome message - `Welcome to Welcome to Ideazz-coalitionAPI`

#### Any background context you want to provide?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
